emoji.js

```diff
@@ -209,32 +219,117 @@ function codeForEmojiStyle(style) {
 	}
 }
 
-const menuIcons = new Map();
+function nativeEmojiLabel() {
+	if (is.macos) {
+		return 'macOS';
```

Comment:
```suggestion
		return 'Apple';
```

This is the "Apple" emoji set (also used on iOS), see the bottom of https://emojipedia.org/